index.html

```diff
@@ -612,7 +640,7 @@ <h3>Consumer</h3>
                     and other UX devices.</p>
                 <figure id="smart-home">
                     <img src="images/wot-use-cases/smart-home.png"
-                        class="wot-arch-diagram"/>
+                        class="wot-arch-diagram"/ alt="smart home use case" />
```

Comment:
The slash before `alt` has to go.

Reply:
fixed.

Reply:
Good, this was the main issue -- approved changes